src/spectrecoff/Prompt.fs

```diff
@@ -3,17 +3,56 @@ module SpectreCoff.Prompt
 
 open Spectre.Console 
 
-let selectionPrompt question choices = 
-    let prompt = SelectionPrompt()
-    prompt.AddChoices (choices |> Seq.toArray) |> ignore
-    prompt.Title <- question
-    prompt
+type PromptOptions = 
+    { Secret: bool
+      Optional: bool }
 
-let prompt prompter =
+let defaultOptions = { Secret = false; Optional = false}
+
+[<RequireQualifiedAccess>]
+module Prompts = 
```

Comment:
Why did you choose to go this way instead of making these private methods for example? Now these "inner" functions are still exposed to the outside, even if you make them put the module in front.

Reply:
Good point, added a private modifier.